WindowsInstaller/config_template.txt

```diff
@@ -81,108 +40,299 @@ allow_user_actions_while_paused = false-----------------------------------------
--- This specifies which display method is set for warmth colours by default. 
--- Possible values: 1 (Red), 2 (Blue Green Red) and 3 (Yellow Orange Red).
-warmth_colors_display_default = 1 
--------------------------------------------------------------------------------
+--
+track_fps = false
 
--------------------------------------------------------------------------------
--- Font file setting.
--- Specify a font file here if you wish to play the game in a language not
--- present in the original game. Examples include Russian, Chinese and Polish.
--- unicode_font = [[X:\ThemeHospital\font.ttc]] 
--------------------------------------------------------------------------------
+---------------------------------------------------------------------------------------------------
+-- Zoom Speed: By default this is set at 80
+-- Any number value between 10 and 1000, 10 is very slow and 1000 is very fast!
+--
+zoom_speed = 80
 
--------------------------------------------------------------------------------
--- Movie global on/off switch.
--- Note that movies will also be disabled if CorsixTH was compiled without the
--- FFMPEG library.
-movies = true 
--------------------------------------------------------------------------------
+---------------------------------------------------------------------------------------------------
+-- Scroll Speeds: The speed of scrolling with and without shift being held.
+-- Any number value between 1 and 10, 1 is very slow and 10 is fast!
+--
+scroll_speed = 2
+shift_scroll_speed = 4
 
--------------------------------------------------------------------------------
+---------------------------------------------------------------------------------------------------
 -- Room information dialogs: Information about new rooms, important for
 -- additional rooms in later levels. Affects campaign only.
+--
 room_information_dialogs = true
--------------------------------------------------------------------------------
 
--------------------------------------------------------------------------------
+---------------------------------------------------------------------------------------------------
 -- If true, parts of the hospital can be made inaccessible by blocking the path
 -- with rooms or objects. If false, all parts of the hospital must be kept
 -- accessible, the game will disallow any attempt to blocking the path.
+--
 allow_blocking_off_areas = false
--------------------------------------------------------------------------------
+
+---------------------------------------------------------------------------------------------------
+-- Direct Zoom: Avoid rendering to an intermediate texture when zooming.
+-- Improves performance and reliability on some hardware.
+--
+direct_zoom = nil
+
+---------------------------------------------------------------------------------------------------
+
+------------------------------------------------ CAMPAIGN MENU ------------------------------------
```

Comment:
I'd like to think I'm at least moderately familiar with CorsixTH and it's various config settings / menus, but the instructions in this CAMPAIGN MENU section don't make any sense to me. I suspect for a person less familiar they might even be actively harmful.

The name itself has to go in a player_name property which we never call out. We don't tell the user that by campaign menu we mean a new game, not the custom campaign selection. The idea was probably that player_name would be injected below, but that doesn't happen either in the template, or if you start the game and quit without setting the name.

suggest tweaking `CorsixTH/Lua/dialogs/resizables/new_game.lua:L102 to ignore blank names, add `player_name = ''` to the default config and put `player_name=[[' .. + config_values.player_name .. ]]` here.

Not keen on the title either, we don't have a campaign menu.